+jrclust/+sort/@TemplateClustering/mergeBySim.m

```diff
@@ -1,5 +1,42 @@
 function nMerged = mergeBySim(obj)
-    %MERGEBYSIM
-    nMerged = 0;
+
```

Comment:
This one is also (almost) identical to `DensityPeakClustering.mergeBySim`. I suggest moving them both to `Clustering`.